styles/components/App.less

```diff
@@ -15,12 +15,16 @@
 
 .App {
   height: 100%;
+  display: flex;
+  flex-direction: column;
```

Comment:
Seems like a mixin for the combo of these two props would be good, but maybe not now but rather as part of the CSS modules port/single resp refactor?